modules/ROOT/images/letItBe.adoc

```diff
@@ -0,0 +1 @@
+
```

Comment:
this file should be removed
if you want to keep the directory, the convention is to create an empty `.gitkeep` file instead.

Reply:
thanks for the tip